config/locales/en.yml

```diff
@@ -122,17 +118,17 @@ en:
         hint: "For example: https://www.gov.uk/pay-vat or /bank-holidays"
   guidance:
     new_step_by_step:
-      shared:
-        title:
-          Add step by step after your title unless it will be inappropriate or make the title excessively long. Keep it under 85 characters
-        slug:
-          "Slug is the link for your step by step. It should be lowercase and you should use hyphens to separate words. For example: get-driving-license"
-        introduction:
-          The introduction explains what the step by step helps the user do, or what it is. Keep it short. Tell users if it is not relevant to them. Do not include any important information in here that doesn’t exist elsewhere.
-        meta_description:
-          Meta description is shown in the GOV.UK search results. It should explain what the step by step helps the user do, or what it is. Keep it under 160 characters.
+      header: New step by step
+      title:
+        Add step by step after your title unless it will be inappropriate or make the title excessively long. Keep it under 85 characters
+      slug:
+        "Slug is the link for your step by step. It should be lowercase and you should use hyphens to separate words. For example: get-driving-license"
+      introduction:
+        The introduction explains what the step by step helps the user do, or what it is. Keep it short. Tell users if it is not relevant to them. Do not include any important information in here that doesn’t exist elsewhere.
+      meta_description:
+        Meta description is shown in the GOV.UK search results. It should explain what the step by step helps the user do, or what it is. Keep it under 160 characters.
     new_step:
-      shared:
+        shared:
```

Comment:
Is the `shared` key still needed?

Reply:
I'm removing it in the next commit, when I'm changing the `new_step` part. It got picked up by diff because I accidentally add a space - will fix that.